### ISSUE TITLE
Isn't Google Map Street View available in Flutter?

### ISSUE THREAD

**Issue by spider-hand** [flutter/flutter#33048]
Hello

Is Google Map Street View available in Flutter? If not is there a plan to provide the plugin in the future? I really want to integrate Street View into my app.

Thanks.

**Comment by azizk-araneux** [comment]
> Hello
> 
> Is Google Map Street View available in Flutter? If not is there a plan to provide the plugin in the future? I really want to integrate Street View into my app.
> 
> Thanks.

@BondarenkoStas Will there be any release soon?

**Comment by daddym85** [comment]
Hi,
any news about streetview plugin?
Thanks.

**Comment by azizk-araneux** [comment]
@daddym85 Unfortunately no news yet. But, I needed that urgently so I developed my own plugin.

**Comment by daddym85** [comment]
@azizk-araneux , is your plugin available?
Thanks

**Comment by azizk-araneux** [comment]
> @azizk-araneux , is your plugin available?
> Thanks

No, because it is not publish ready as I just created it for myself and it doesn't even fulfill the pub.dev's guidelines.

**Comment by pelli23** [comment]
@azizk-araneux Could you share your plugin code please?

**Comment by azizk-araneux** [comment]
I can share you the base code? Will that be okay?

**Comment by pelli23** [comment]
Yes, of course. It would be great.

Thanks.

**Comment by viicentecs** [comment]
@azizk-araneux  Could you share your plugin code please?

**Comment by daddym85** [comment]
Hello there ,
 any news about the flutter plugin for  streetview?

**Comment by abdullah432** [comment]
I think you can do this. Try below code
```
    GoogleMap(
      ...
     mapType: MapType.normal,
    ),
```

**Comment by anurag-tekale** [comment]
@azizk-araneux Share street view plugin please

**Comment by anurag-tekale** [comment]
@pelli23 Could u share street view plugin please ?

**Comment by ericwoerner** [comment]
Is there any update on this item?

**Comment by adnanefouam** [comment]
any plugin yet ?

**Comment by ericwoerner** [comment]
> @daddym85 Unfortunately no news yet. But, I needed that urgently so I developed my own plugin.

Do you mind sharing your code?

**Comment by valterh4ck3r** [comment]
> > @azizk-araneux , is your plugin available?
> > Thanks
> 
> No, because it is not publish ready as I just created it for myself and it doesn't even fulfill the pub.dev's guidelines.

Do you pretend publish your plugin ?

**Comment by adnanefouam** [comment]
![95F233F3-B494-4488-9F4E-B37DAFEEB662](https://user-images.githubusercontent.com/59290932/93122046-c30ecb80-f6bd-11ea-9401-93d38f6d3ae5.png)

guys we can use just webview for this senario or native code ,for me webview worked perfectly whenever i want streetview i passe latitude longtitude data to the javascript function

**Comment by tobiaslewinzon** [comment]
> ![95F233F3-B494-4488-9F4E-B37DAFEEB662](https://user-images.githubusercontent.com/59290932/93122046-c30ecb80-f6bd-11ea-9401-93d38f6d3ae5.png)
> 
> guys we can use just webview for this senario or native code ,for me webview worked perfectly whenever i want streetview i passe latitude longtitude data to the javascript function

Is this allowed with no limitations on how many calls can be done?

**Comment by tobiaslewinzon** [comment]
Has anyone tried using the SDKs for iOS and Android and run them as native code?

**Comment by ahmedcs2600** [comment]
https://github.com/ahmedcs2600/Flutter-StreetView-

I have implemented native code(currently supports Android only for now).
You guys can use native implementation if it's helpful to you. Feel free to contribute and ask me for help :)

**Comment by Vito2089** [comment]
@google-admin any news on this???

**Comment by radikris** [comment]
> ![95F233F3-B494-4488-9F4E-B37DAFEEB662](https://user-images.githubusercontent.com/59290932/93122046-c30ecb80-f6bd-11ea-9401-93d38f6d3ae5.png)
> 
> guys we can use just webview for this senario or native code ,for me webview worked perfectly whenever i want streetview i passe latitude longtitude data to the javascript function

Hi, can you show me source where can I find this solution, it looks good for me

**Comment by Nidria** [comment]
Anything yet?

**Comment by zyzdev** [comment]
Hello everyone,

Flutter plugin [flutter_google_street_view](https://pub.dev/packages/flutter_google_street_view) provides street view and support web, Android and iOS.
Try it!

**Comment by valterh4ck3r** [comment]
> Hello everyone,
> 
> Flutter plugin [flutter_google_street_view](https://pub.dev/packages/flutter_google_street_view) provides street view and support web, Android and iOS.
> Try it!

Perfect.
